tools/jit/gen_jit_dispatch.py

```diff
@@ -46,16 +46,20 @@
 """)
 
 CALL_NAMESPACE = CodeTemplate("""\
-AutoGPU device_guard(deviceForInputs(stack, ${num_dynamic_inputs}));
 auto result = at::${name}(${args});
 """)
 CALL_METHOD = CodeTemplate("""\
 AutoGPU device_guard(deviceForInputs(stack, ${num_dynamic_inputs}));
```

Comment:
Is this still needed or do the ATen ops now correctly set the device on their own?